_drafts/submitting-your-first-pull-request.markdown

```diff
@@ -0,0 +1,205 @@
+---
+layout: post
+title: Submitting Your First Pull Request
+author: dave_paquette
+---
+
+Over the last few years, we have seen a big shift in the .NET community towards open source. In addition to a huge number of open source community led projects, we have also seen Microsoft move major portions of the .NET framework over to GitHub. 
+
+With all these packages out in the wild, the opportunities to contribute are endless. The process however can be a little daunting for first timers, especially if you are not using git in your day-to-day work. In this post I will guide you through the process of submitting your first pull request. I will be showing examples from my experience contributing to the [Humanitarian Toolbox](http://www.htbox.org/)’s [allReady project](https://github.com/HTBox/allReady). As with all things git related, there is more than one way to do everything. This post will outline the workflow I have been using and should serve as a good starting point for most .NET developers who are interested in getting started with open source projects hosted on GitHub.
```

Comment:
Suggestion: Change "I will be showing" to "I will show" to make it more active (i.e. less Canadian)